microcosm_elasticsearch/factories.py

```diff
@@ -5,13 +5,29 @@
 from os import environ
 
 import boto3
```

Comment:
small comment about import order - we follow PEP 8 imports style guide (https://www.python.org/dev/peps/pep-0008/#imports) for grouping imports by stdlib, third-party, package, so would typically be:

```
from os import environ
from collections import namedtuple
from functools import partial

import boto3
from elasticsearch import Elasticsearch, RequestsHttpConnection
from microcosm.api import defaults
from requests_aws4auth import AWS4Auth

from microcosm_elasticsearch.serialization import JSONSerializerPython2
```